tests/src/Mod/Part/App/TopoShapeMakeShapeWithElementMap.cpp

```diff
@@ -59,3 +60,61 @@ TEST_F(TopoShapeMakeShapeWithElementMapTests, nullShapeThrows)
     EXPECT_THROW(Shape()->makeShapeWithElementMap(nullShape, *Mapper(), sources),
                  Part::NullShapeException);
 }
+
+using Data::IndexedName, Data::MappedName;
+using Part::TopoShape;
+
+// TEST_F(TopoShapeMakeShapeWithElementMapTests, mapVertex)
+// TEST_F(TopoShapeMakeShapeWithElementMapTests, mapEdge)
+// TEST_F(TopoShapeMakeShapeWithElementMapTests, mapWire)
+// TEST_F(TopoShapeMakeShapeWithElementMapTests, mapFace)
+// TEST_F(TopoShapeMakeShapeWithElementMapTests, mapShell)
+// TEST_F(TopoShapeMakeShapeWithElementMapTests, mapSolid)
+
+TEST_F(TopoShapeMakeShapeWithElementMapTests, mapCompound)
```

Comment:
I hate to see a test with a cognitive complexity so high -- is there a logical seam in here someplace?